src/model/channel/message.rs

```diff
@@ -326,9 +326,15 @@ impl Message {
     where
         F: for<'b> FnOnce(&'b mut EditMessage<'a>) -> &'b mut EditMessage<'a>,
     {
+        let mut builder = self._prepare_edit_builder(cache_http.cache())?;
```

Comment:
```suggestion
        let mut builder = self._prepare_edit_builder()?;
```

Reply:
As it stands this requires a feature gate on the `cache` feature, which doesn't work, since the function contains code that isn't gated. I recommend instead putting the `#[cft(feature = "cache")]` block back into `Message::edit`, before calling `_prepare_edit_builder`. I don't think it'll add too much compile-time bloat, though do note that the `cache` feature is enabled by default, so you may want to perform some benchmarks.

Reply:
solved!